docker_containers/mindmeld_docker/Dockerfile

```diff
@@ -90,4 +93,5 @@ ENTRYPOINT export LC_ALL=C.UTF-8 && \
     pip3 install mindmeld --upgrade && \
     su mindmeld -c "ES_JAVA_OPTS='-Xms1g -Xmx1g' /elasticsearch/bin/elasticsearch -d" && \
     mindmeld num-parse --start && \
+    /waitForClusterToBeGreen.sh && \
```

Comment:
Change the file name to `wait_for_cluster_to_turn_green.sh` to reflect the snake case of the other file names in the repo

Reply:
Done